### PR TITLE
Replacing Nunjucks dep with one that works with html-webpack-plugin

### LINKED ISSUE #135: You have to restart the dev server after editing an HTML file
When you edit an HTML file, you have to `CTRL+C` and restart the dev server (and then manually reload your browser after the dev server has booted up).

[Trello ticket](https://trello.com/c/R74bQwbq)

### PR DESCRIPTION
**Pull request by aendrew**
Fixes #135.

### CODE REVIEW

File: views/layout.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
+<!doctype html>
```
Comment (callumlocke) [inline views/layout.html]: wow was this missing?

File: webpack.config.babel.js [hunk 2]
```diff
@@ -74,25 +85,17 @@ module.exports = async (env = 'development') => ({
     ],
   },
   devServer: {
-    hot: true,
-    contentBase: resolve(__dirname, 'client'),
+    hot: false,
```
Comment (callumlocke) [inline webpack.config.babel.js]: does this mean HMR is now off for JS modules?

File: package.json [hunk 3]
```diff
@@ -73,7 +76,7 @@
     "prebuild": "npm run clean",
     "pretest": "npm run build",
     "qa": "mocha ./test/**/*.spec.js",
-    "start": "webpack-dev-server"
+    "start": "webpack-dev-server -w"
```
Comment (callumlocke) [inline package.json]: personal preference, I think it's better to spell out flags like `--watch` when defining commands in npm scripts

File: views/filters/index.js [hunk 4]
```diff
@@ -48,7 +48,7 @@ const markdown = markdownIt({
 
 export function md(str, inline) {
   return !str ? '' :
-    new SafeString(inline ? markdown.renderInline(str) : markdown.render(str));
+    inline ? markdown.renderInline(str) : markdown.render(str);
```
Comment (callumlocke) [inline views/filters/index.js]: why was this change necessary? just curious

File: package.json [hunk 5]
```diff
@@ -44,9 +48,8 @@
     "node-sass": "^4.5.3",
     "nodemon": "^1.11.0",
     "nunjucks": "^3.0.1",
-    "nunjucks-loader": "^2.4.8",
+    "nunjucks-html-loader": "github:ft-interactive/nunjucks-html-loader",
```
Comment (callumlocke) [inline package.json]: i guess forking is ok for the time being, as long as we move away from nunjucks soonish

File: package.json [hunk 6]
```diff
@@ -35,7 +35,11 @@
     "eslint-plugin-jsx-a11y": "^5.1.1",
     "eslint-plugin-react": "^7.1.0",
     "extract-text-webpack-plugin": "^3.0.0",
+    "file-loader": "^0.11.2",
     "ft-graphics-deploy": "^2.1.2",
+    "html-loader": "^0.5.1",
+    "html-webpack-harddisk-plugin": "^0.1.0",
```
Comment (callumlocke) [inline package.json]: what does this do? can't see it referenced anywhere else

**Comment by ajwl** [comment]
👍  code works, LGTM

File: views/layout.html (hunk 1 above)
Comment (aendrew) [inline views/layout.html]: No, it was beneath two Nunjucks directives, which was causing newlines before the doctype. I figured it's probably better to not do that...

File: webpack.config.babel.js (hunk 2 above)
Comment (aendrew) [inline webpack.config.babel.js]: Yes, by default, though it's easy enough to re-enable. Hot reloading needs to be off to force a page reload after Nunjucks template update, otherwise it needs to be manually refreshed.

I figure it's an acceptable trade-off while we're still in a world where people do a lot of HTML template editing.

File: package.json (hunk 6 above)
Comment (aendrew) [inline package.json]: `html-loader` wraps the Nunjucks output in a module that can be consumed by `html-webpack-plugin`. `html-webpack-harddisk-plugin` writes to disk on compilation, but in hindsight I may not have actually added it to the Webpack config — good catch!

File: package.json (hunk 5 above)
Comment (aendrew) [inline package.json]: Yeah, agreed. It's unfortunate that pretty much all Nunjucks plugins for Webpack are kind of rubbish.

File: package.json (hunk 3 above)
Comment (aendrew) [inline package.json]: Fair point, will update.

File: views/filters/index.js (hunk 4 above)
Comment (aendrew) [inline views/filters/index.js]: I'm not super sure, TBTH. When I was wrapping it as a SafeString, it was double-escaping things, so the rendered HTML from Markdown was showing up in the body text.
